### ISSUE TITLE
release for kinecik

### ISSUE THREAD

**Issue by tulku** [RIVeR-Lab/apriltags_ros#10]
Hi!

Sorry to bother you, but are you planning on making a release for ros kinetic?

Thank you very much in advance!

**Comment by ab3nd** [comment]
I'm not one of the devs, but I'm using it with Kinetic (git clone & build with catkin_make) on Ubuntu 16.04, and so far haven't had any major issues. I am running into the lack of synchronization mentioned in issue #13, though.

**Comment by AlexisTM** [comment]
You can simply build it.

**Comment by v4hn** [comment]
> You can simply build it.

Yes, but that does not help you if you plan to release packages that depend on this one.
Also it forces you to create another workspace - or drag it into every workspace where you want to use it.

If at all possible, please finally release this into kinetic!

**Comment by AlexisTM** [comment]
Not at all, you can simply build it with the install command, and you set the install path to /opt/ros/kinetic : solved

**Comment by velinddimitrov** [comment]
We are in the process of looking at a better implementation, maybe AprilTags2 to wrap, and improve performance. We are moving robots slowly to Kinetic, so I would expect a Kinetic release in the coming month or two.

**Comment by v4hn** [comment]
@AlexisTM :

1. Untracked files do not belong there and will break in the long run. It's a tedious job to debug such problems. Apparently you never had to.

2. And you do that on *every single machine* of your end-user. This is called packaging.
Usually people do that *once per distribution* instead of *once per end-user*

3. If you like working with LFS, go for it. The rest of us uses package build/distribution systems.

@velinddimitrov 

Thank you a lot for your feedback! We worked around our issues for the moment, but this comes up over and over again... I had no idea that you would like to replace the underlying library for kinetic. I'm very much looking forward to an improved kinetic-devel branch and a package release!

**Comment by v4hn** [comment]
@velinddimitrov ping 

Has there been any progress on apriltags2 for this package?
Have there been any incomplete attempts you are aware of?
It would still be very nice to see this in kinetic, even more so if it uses the apriltags2 lib.

**Comment by AlexisTM** [comment]
@v4hn Nobody said you are not allowed to use bloom to package ;)

**Comment by v4hn** [comment]
@AlexisTM Nobody said I'm not allowed to spend work on getting it to work with apriltags2 either.
I asked for a status update though.
I *do* believe it is a good idea to move to apriltags2 before a(n eventual) release.

apriltags_ros is no [orphaned package](https://github.com/ros-infrastructure/rep/pull/150) and I am very happy about that.
